tests/register.spec.ts

```diff
@@ -19,6 +20,7 @@ const userLastName = faker.person.lastName();
 const userEmail = `${userFirstName}.${userLastName}@example.com`.toLowerCase();
 const userPassword = "Password1!";
 const userGender = "Male";
```

Comment:
Please add a TODO comment that this and the other questions need to be matched to mock data (which in turn needs to be stored as json or TS to be read in.